Support docs to docs site IDE freeze large DAG (#8980)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Troubleshooting/ide-freezes-lineage-tab-opened.md

---
title: "Studio IDE freezes when opening the Lineage tab"
description: "Workarounds for Studio IDE freezing or becoming unresponsive when the Lineage tab is active, especially with large models or multiple open model tabs."
sidebar_label: "Studio IDE freezes when opening the Lineage tab"
id: studio-ide-freezes-lineage-tab
---

If the <Constant name="studio_ide" /> freezes with two or more models open and the **Lineage** tab active, use the following workarounds.

### Why this happens

> The <Constant name="studio_ide" /> can freeze when the **Lineage** tab is open or loads automatically, and when two or more models are open, especially if one model has a large or slow-loading <Term id="dag" />.

Author: ```suggestion
The <Constant name="studio_ide" /> can freeze when the **Lineage** tab is open or loads automatically, and when two or more models are open, especially if a model has a large or slow-loading <Term id="dag" />.
```

6. Refresh the page and sign in again if prompted.

### Mozilla Firefox

1. Open the <Constant name="dbt_platform" /> page that’s freezing.
2. Open DevTools (right-click the page and select **Inspect**).
3. Open the **Storage** tab.
   - If you don’t see it, open DevTools **Settings** and enable **Storage**.
4. In the left sidebar, expand **Local Storage**.
5. Right-click the origin that matches your current URL (for example, `https://ab123.us1.dbt.com`) and select **Delete All**.
6. Refresh the page and sign in again if prompted.

### Safari

1. Open Safari and go to the <Constant name="dbt_platform" /> page that’s freezing.
2. If needed, enable developer features:
   - Go to **Safari** > **Settings**.
   - Click **Advanced**.
   - Enable developer features.
3. Open Web Inspector from **Develop** > **Show Web Inspector**.
4. Open the **Storage** tab.
5. In the left sidebar, expand **Local Storage** and select the origin that matches your current URL (for example, `https://ab123.us1.dbt.com`).
6. Delete the Local Storage entries for that origin.
7. Refresh the page and sign in again if prompted.

## Need more help?

> If freezing continues after trying these workarounds, [contact dbt Support](mailto:support@getdbt.com) and include:

Reviewer: <img width="716" height="256" alt="Screenshot 2026-04-17 at 12 46 05" src="https://github.com/user-attachments/assets/9868b3df-2fad-40f3-b0e6-186a1bfb2bd8" />

the bullets aren't lined up correctly

- A HAR file (refer to [How to generate HAR files](/faqs/Troubleshooting/generate-har-file))

>   - Your account URL

Author: ```suggestion
- Your account URL
```

>   - Browser and version

Author: ```suggestion
- Browser and version
```

>   - Approximate number of open model tabs

Author: ```suggestion
- Approximate number of open model tabs
```

>   - Whether the issue happens only when the **Lineage** tab is active

Author: ```suggestion
- Whether the issue happens only when the **Lineage** tab is active
```

---
title: "Studio IDE freezes when opening the Lineage tab"
description: "Workarounds for Studio IDE freezing or becoming unresponsive when the Lineage tab is active, especially with large models or multiple open model tabs."
sidebar_label: "Studio IDE freezes when opening the Lineage tab"
id: studio-ide-freezes-lineage-tab
---

> If the <Constant name="studio_ide" /> freezes or becomes unresponsive when you have two or more models open and the **Lineage** tab is active, use the workarounds in this guide.

Reviewer: ```suggestion
If the <Constant name="studio_ide" /> freezes with two or more models open and the Lineage tab active, use the following workarounds.

### Why this happens
The <Constant name="studio_ide"/>  can freeze when you have the **Lineage tab** open or it loads automatically, and when two or more models are open, especially if any model has a large or slow-loading <Term id="dag"/>.
```

Reviewer: sugg simplifying the sentences so it's easier to parse through. if you accept the above, you'll want to remove the common reasons header + content

## Common reasons

The <Constant name="studio_ide" /> may freeze when:
- You have two or more models open, and
- The **Lineage** tab is open or loads automatically.

This is more likely when at least one model is large and the DAG takes longer to load.

## What you might observe

> - The editor becomes unresponsive for a period of time.

Reviewer: period time is a bit vague - can we elaborate or just be direct and keep it to period of time

- After a few minutes, your browser may display a **Wait** or **Exit** dialog.

## Workaround 1: Avoid the Lineage tab on startup

1. Refresh the page.
2. Keep the console focused on another tab (for example, **Results** or **Compiled code**) instead of **Lineage**.

> 3. Open **Lineage** only when needed, and consider closing extra model tabs first.

Reviewer: this step is doing two things. i recommend splitting into separate steps or pick one instruction — "close extra model tabs" and "open Lineage only when needed" are distinct actions.

## Workaround 2: Clear Local Storage for your account URL

This workaround is browser-dependent. In DevTools, select the Local Storage origin that matches the URL in your browser address bar. For example, your account access URL might look like `https://ab123.us1.dbt.com`.

> ### Google Chrome

Reviewer: great stuff here! chrome and edge steps are almost identical. suggest collapsing  them into one section: Chrome and Edge, and note any edge-specific differences only if they exist. having 4 near-duplicate sections is a maintenance burden.

6. Refresh the page and sign in again if prompted.

### Mozilla Firefox

1. Open the <Constant name="dbt_platform" /> page that’s freezing.
2. Open DevTools (right-click the page and select **Inspect**).
3. Open the **Storage** tab.
   - If you don’t see it, open DevTools **Settings** and enable **Storage**.
4. In the left sidebar, expand **Local Storage**.
5. Right-click the origin that matches your current URL (for example, `https://ab123.us1.dbt.com`) and select **Delete All**.
6. Refresh the page and sign in again if prompted.

### Safari

1. Open Safari and go to the <Constant name="dbt_platform" /> page that’s freezing.
2. If needed, enable developer features:
   - Go to **Safari** > **Settings**.
   - Click **Advanced**.
   - Enable developer features.
3. Open Web Inspector from **Develop** > **Show Web Inspector**.
4. Open the **Storage** tab.
5. In the left sidebar, expand **Local Storage** and select the origin that matches your current URL (for example, `https://ab123.us1.dbt.com`).
6. Delete the Local Storage entries for that origin.
7. Refresh the page and sign in again if prompted.

> ## If the issue persists

Reviewer: issue persists is a bit vague and jargon-y - sugg being more clear adn direct
```suggestion
## Need more help?
```

> If you still see freezing after trying the workarounds:

Reviewer: we're not really giving users direct instructions
```suggestion
If freezing continues after trying these workarounds, [contact dbt Support](mailto:support@getdbt.com) and include:
```

> - Capture a HAR file and share it with [dbt support](/docs/dbt-support):

Reviewer: since we're sharing all the below to support we might as well consolidate this so it's cleaner
```suggestion
- A HAR file (refer to [How to generate HAR files](/faqs/Troubleshooting/generate-har-file))
```

>   - Refer to [How to generate HAR files](/faqs/Troubleshooting/generate-har-file)

Reviewer: ```suggestion
```

> - Contact Support and include:

Reviewer: since we're remove the duplicate 'contact support, the rest can remain
```suggestion
```

  - Your account URL

>   - Browser + version

Reviewer: ```suggestion
  - Browser and version
```
